pom.xml

```diff
@@ -0,0 +1,40 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>ua.lviv.iot</groupId>
+  <artifactId>officeSuppliesShopProject</artifactId>
+  <packaging>jar</packaging>
+  <version>1.0-SNAPSHOT</version>
+  <name>officeSuppliesShopProject</name>
+  <url>http://maven.apache.org</url>
+  
+ <properties>
+    <maven.compiler.release>17</maven.compiler.release>
+  </properties>
+  <dependencies>
+    <dependency>
+      <groupId>org.projectlombok</groupId>
+      <artifactId>lombok</artifactId>
+      <version>1.18.24</version>
+      <scope>provided</scope>
+    </dependency>
```

Comment:
You have Lombok in your project, but I can't find you using it anywhere. If it's just for the demonstration of dependency usage in pom.xml, it's ok. Otherwise better not add dependencies you're not using - it takes longer to build your project.